Extension/config.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
+      rel="icon"
+      href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🖖</text></svg>"
+    />
+
+    <script
+      type="text/javascript"
+      src="https://extension-files.twitch.tv/helper/v1/twitch-ext.min.js"
+    ></script>
+
+    <title>Project Pony</title>
```

Comment:
Should the title be Project Pony? This was just a custom extension we had built and isn't actually Pony/Gateway.

Reply:
Good catch. Total copy/paste error.